.scalafmt.conf

```
@@ -9,8 +9,7 @@ newlines.source = keep
 
 # http://docs.scala-lang.org/style/scaladoc.html recommends Asterisk style (aka JavaDoc style)
 # scala/scala is written that way too https://github.com/scala/scala/blob/v2.12.2/src/library/scala/Predef.scala
-# TODO: switch to Asterisk
-docstrings.style = keep
+docstrings.style = Asterisk
```

```suggestion
docstrings.style = Asterisk
docstrings.wrap = no
```

I am annoyed with each Scalafmt version randomly contaminating changes into git history long after Scalafmt was first introduced. I am often too lazy to find out the backward compatible configuration incantation to keep the source code as-is, but `docstrings.wrap = no` I think is necessary because the change it makes is not only unnecessary, but often unergonomic, like turning neatly ordered list into:

```
1.
1.
1.
```

---

Okay I dropped that commit.